src/softwarelist.cpp

```diff
@@ -1012,6 +1012,7 @@ QString &SoftwareList::lookupMountDevice(QString device, QString deviceInterface
 	QStringList xmlLines(qmc2MachineList->xmlDb()->xml(systemName).split('\n', QString::SkipEmptyParts));
 	QStringList *xmlData = &xmlLines;
```

Comment:
This should ideally go away, but I was not sure how to replace the routine in lines 1044-1045. Additionally, it appears that this code does not ever gets triggered in the current codebase, making it difficult to test.